bin/cli.js

```diff
@@ -308,3 +311,19 @@ function displayHelp() {
   const usage = commandLineUsage(sections);
   console.log(usage);
 }
+
+function updateRuntime() {
+  const installRuntime = require('./install-runtime');
+  cli.spinner('  Updating runtime…');
+  installRuntime()
+  .then(() => {
+    cli.spinner(chalk.green.bold('✓ ') + 'Updating runtime… done!', true);
+  })
+  .catch(error => {
+    cli.spinner(chalk.red.bold('✗ ') + 'Updating runtime… failed!', true);
+    console.error(error);
+  })
+  .finally(() => {
+    process.exit();
```

Comment:
Good question! I suppose it depends on whether the command succeeds or fails. I've made that change in fa2bb02.